src/components/publishing/display/canvas/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react"
+import styled, { StyledFunction } from "styled-components"
+import Colors from "../../../../assets/colors"
+import { pMedia } from "../../../helpers"
+import Fonts from "../../fonts"
+import Canvas from "./container"
```

Comment:
Ya I think this is another instance where the naming is a little confusing. Canvas and container seem like two different things when they are not.